src/css/base/_float-box.scss

```diff
@@ -0,0 +1,9 @@
+.float-box {
```

Comment:
`base` folder should only have unclassed HTML elements. Class based styling should be moved to `object` or `components`.